config/manager/manager.yaml

```diff
@@ -97,7 +97,7 @@ spec:
         resources:
           limits:
             cpu: 500m
-            memory: 128Mi
+            memory: 500Mi
```

Comment:
Did you find that with no PVC, the added files do in fact get included in this memory calculation? Is that why you adjusted it?

Reply:
Sort off, while testing I found out that pod was crashing with OOM and seen this issue opened already https://github.com/src-d/go-git/issues/447, that's the reason I have increased it

Reply:
I wonder if we want to potentially remove the limit altogether here, and assume anyone who kustomizes outside of this file will set it to whatever they need (if they set it at all)

Reply:
Yes, while cloning it's consumes more memory

Reply:
I saw we remove this field. What do you think?

Reply:
sure, we can remove it